draft-ietf-scitt-architecture.md

```diff
@@ -547,18 +547,17 @@ It indicates the Signed Statement is securing a JSON content type, and identifyi
 
 To register a Signed Statement, the Transparency Service performs the following steps:
 
-1. **Client authentication:** A Client authenticates with the Transparency Service, to Register Signed Statements on behalf of one or more Issuers.
-Authentication and authorization is implementation-specific, and out of scope of the SCITT Architecture.
-1. **Issuer Verification:** The Transparency Service MUST syntactically validate the Issuer's identity Claims, which may be different than the Client identity.
+1. **Client authentication:** A Client authenticates the Transparency Service before registering Signed Statements on behalf of one or more Issuers.
+Authentication and authorization is implementation-specific, and out of scope of the SCITT architecture.
+1. **Issuer Verification:** The Transparency Service MUST validate the Issuer's identity Claims, which may be different than the Client identity.
```

Comment:
```suggestion
1. **Issuer Verification:** The Transparency Service MUST syntactically verify the Issuer's identity Claims, which may be different than the Client identity. Only information local to the signed statement need be considered.
```

"validate the Issuer's identity Claims" remains ambiguous to me. I prefer that registration policies are the vehicle for ts's to express different, use case based, constraints on what is accepted on a ledger. If we must have a 'minimum' bar, verification of claims based exclusively on the local contents of the signed statement is what I support.

Reply:
```suggestion
1. **Issuer Verification:** The Transparency Service MUST perform signature verification, as defined in [RFC 9052 4.4](https://datatracker.ietf.org/doc/html/rfc9052#section-4.4), and MAY perform additional checks as part of its Registration Policy.
```

Reply:
That is much better. THanks